### PR TITLE
Add the sha of the the current commit to the watermark as the fragment of the watermark image url.

### PR DESCRIPTION
**Pull request by tasdomas**
This PR also updates the way updatable comments are determined - the matcher should ignore the sha in
the watermark.

This PR supercedes #1304.

### CODE REVIEW

File: src/cml.js [hunk 1]
```diff
@@ -294,7 +276,7 @@ class CML {
     let comment;
     const updatableComment = (comments) => {
       return comments.reverse().find(({ body }) => {
-        return body.includes(watermark);
+        return watermark === null || watermark.isIn(body);
```
Comment (DavidGOrtega) [inline src/cml.js:279]: silly nitpick just in case is `undefined`
```suggestion
        return !watermark || watermark.isIn(body);
```

**Review by DavidGOrtega — CHANGES_REQUESTED** [review]
Just only a little observation that might be even incorrect

File: src/cml.js (hunk 1 above)
Comment (tasdomas) [reply src/cml.js]: That's a good catch - thank you!

Comment (tasdomas) [reply src/cml.js]: Addressed in https://github.com/iterative/cml/pull/1309/commits/84403766e70647da2175ce93af6629d1df322ccc

**Review by dacbd — APPROVED** [review]
I like it more than mine.
https://github.com/iterative/cml-playground/pull/294

**Review by DavidGOrtega — APPROVED** [review]
lgtm 🚀

**Pull request merged**